cli/pkg/kubectl/base/kubeconfig.go

```diff
@@ -137,3 +138,12 @@ func EnsureKubeconfigSecret(ctx context.Context, kubeconfig, name string, client
 
 	return secret, false, nil
 }
+
+// LoadRestConfigFromFile loads a kubeconfig string and returns a rest.Config
+func LoadRestConfigFromFile(kubeconfig string) (*rest.Config, error) {
```

Comment:
Shoulds this be named `Load...FromFile`? If I see that correct in the use the `kubeconfig` input is supposed to be the content of a kubeconfig file, not the path to a file.

I think something like `ParseKubeconfig` or something would be more fitting.